### PR TITLE
move thrift client call wrapper to a separate module

### PR DESCRIPTION
**Pull request by gyorb**

### CODE REVIEW

File: libcodechecker/libclient/thrift_call.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    def wrapper(self, *args, **kwargs):
+        # LOG.debug('['+self.__host+':'+str(self.__port)+'] '
+        #       '>>>>> ['+funcName+']')
+        # before = datetime.datetime.now()
+        self.transport.open()
+        func = getattr(self.client, funcName)
+        try:
+            res = func(*args, **kwargs)
+            return res
+        except shared.ttypes.RequestFailed as reqfailure:
```
Comment (csordasmarton) [inline libcodechecker/libclient/thrift_call.py]: Do not we want to handle `API_MISMATCH` here too?
https://github.com/Ericsson/codechecker/blob/98f4ccd541566f43dd358d84321dc3056436b79c/libcodechecker/libclient/authentication_helper.py#L68

File: libcodechecker/libclient/thrift_call.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                LOG.debug('Unknown thrift error')
+            elif ex.type == TProtocolException.INVALID_DATA:
+                LOG.debug('Thrift invalid data error.')
+            elif ex.type == TProtocolException.NEGATIVE_SIZE:
+                LOG.debug('Thrift negative size error.')
+            elif ex.type == TProtocolException.SIZE_LIMIT:
+                LOG.debug('Thrift size limit error.')
+            elif ex.type == TProtocolException.BAD_VERSION:
+                LOG.debug('Thrift bad version error.')
+            LOG.debug(funcName)
```
Comment (csordasmarton) [inline libcodechecker/libclient/thrift_call.py]: Wouldn't it be better to write these log messages as error not debug?

File: libcodechecker/libclient/thrift_call.py (hunk 1 above)
Comment (gyorb) [reply libcodechecker/libclient/thrift_call.py]: done

File: libcodechecker/libclient/thrift_call.py (hunk 2 above)
Comment (gyorb) [reply libcodechecker/libclient/thrift_call.py]: done

**Pull request merged**